### PR TITLE
Fix typo in A14-channelz

### PR DESCRIPTION
**Pull request by veblush**
I just read this doc for the code review and fix some minor typos. Also I removed the pic picture "A14_graphics/4.svg" in the doc because it looks somewhat irrelevant.

### CODE REVIEW

File: A14-channelz.md [hunk 1]
```diff
@@ -285,10 +282,10 @@ Calls may send and receive any number of messages, including zero.
 
 The number of keepalives sent represents how often the local endpoint wants to
 keep the socket alive.  Generally this is an HTTP/2 PING frame.  However, not
-all PING frames are keep alives.  Specifically, pings that were sent as a reply
-(ACK bit set) or were used for reasons unrelated to keep alives should not count
+all PING frames are kept alive.  Specifically, pings that were sent as a reply
```
Comment (carl-mastrangelo) [inline A14-channelz.md]: The original way is correct.  A keep alive is a noun in this context.

Comment (veblush) [reply A14-channelz.md]: Oh Got it.

**Comment by carl-mastrangelo** [comment]
fyi this still shows up as a WIP.

**Comment by veblush** [comment]
Thanks you for the review!

**Pull request merged**